.travis.yml

```diff
@@ -17,12 +17,12 @@ script:
   - python -c 'import opentrons; print(opentrons.__version__)'
   # lint all code
   - flake8 .
-  - pytest protolib/ -vv
+  - pytest protolib2/ -vv
   - python ./scripts/bad-README-subcategory.py # make sure subcategories don't have 2 spaces
   # - ./scripts/smoke-test.sh  # skip smoke testing, hope protolib catches it
   #
-  # use protolib to build zipped .json for S3
-  - python -m protolib protocols/ releases/
+  # use protolib2 to build zipped .json for S3
```

Comment:
This switch shouldn't happen until protolib2 is entirely working, and it should use `make parse-ot1 parse-ot2` commands not `python -m` because it needs to switch between virtualenvs, which the Makefile handles.